### DISCUSSION TITLE
如何实现两个网盘同步

### DISCUSSION THREAD

**Discussion by wjjchen** [AlistGo/alist#2109] [Q&A]
可以直接把一个存储中的文件夹复制到另一个存储吗？

**Comment by imrelax** [comment]
https://alist.nn.ci/zh/guide/advanced/balance.html

貌似官方说明的 均衡负载是可以的。
但是我用的是群晖直接开2个同步！

![qh](https://user-images.githubusercontent.com/33931868/197942788-d1ec55bc-ad50-42db-a24a-584cd0ccb735.png)

**Reply by naicfeng** [reply]
均衡负载。要两个目录文件期初完全一致，不然下载有可能会出现错误。并且只均衡下载，不能自动上传到多个网盘。

**Reply by imrelax** [reply]
那就用群晖。 
我用的是群晖。 同时将一个文件 同步到的 微软 阿里 189等盘。所有盘都保持了同步！！

**Reply by imrelax** [reply]
直接在线复制 估计有难度。 我用的是本地群晖执行。
比如 微软盘有变动。 会自动同步变动到本地。然后本地因为有修改 会自动修改 阿里 189等盘。所有保持了全部网盘数据一致 

当然 我也可以设置部分盘 只新增不下载。或者 只下载不新增 。 类似的功能还有其他同步软件也可以！
